src/main/java/amazin/User.java

```diff
@@ -0,0 +1,85 @@
+package amazin;
+
+import javax.persistence.*;
+
+@Entity
+public class User {
+
+    @Id
+    @GeneratedValue(strategy= GenerationType.AUTO)
+    @Column(name = "id", updatable = false, nullable = false)
+    private Long id = null;
```

Comment:
This could be set to int

Reply:
Actually, it's common for the id to be long because the id's are not resusable so when you delete a record, the id still keeps going up. Eventually, an int won't be enough to store the id field.